build

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
```

Comment:
How can emscripten depend on this? It will run as a subprocess of python/scons rather than in this shell.

Reply:
It is caused by `source emsdk_env.sh`, as that line have been removed, no change is needed now.